src/attr.rs

```diff
@@ -78,58 +83,51 @@ pub(crate) fn parse_our_attr(attr: &Attribute) -> Result<OurAttr, ()> {
     let name = match it.next() {
         Some(TokenTree::Ident(x)) => x,
         Some(other) => {
-            return Spanned::span(&other)
-                .err(format!("expected ident, found '{}'", other))
-                .emit_with_attr_note();
+            emit_error!(other.span(), "expected ident, found '{}'", other);
+            return Err(());
         }
         None => {
-            return attr.tokens.span()
-                .err("expected ident, found nothing")
-                .emit_with_attr_note();
+            emit_error!(attr.tokens.span(), "expected ident, found nothing");
+            return Err(());
         }
     };
 
-    // Extract the parameters (which again, have to be a group delimitted by
+    // Extract the parameters (which again, have to be a group delimited by
     // `()`)
     let params = match it.next() {
         Some(TokenTree::Group(ref g)) if g.delimiter() == Delimiter::Parenthesis => {
             g.stream()
         }
         Some(other) => {
-            let msg = format!(
+            emit_error!(
+                other.span(),
                 "expected arguments for '{}' in parenthesis `()`, found `{}`",
                 name,
-                other,
+                other
```

Comment:
I like my trailing commas

Reply:
Everybody say they like trailing commas, am I the only one weird lunatic here who hates them and eradicates wherever possible? Well, `rustfmt` has such option nonetheless...

Reply:
Wow, I just realized that these macros do not support trailing commas :cry: Let's go fix it first...